commands/addcc.ts

```diff
@@ -0,0 +1,110 @@
+import { SlashCommandBuilder, roleMention } from '@discordjs/builders';
+import { CommandInteraction } from 'discord.js';
+import { createClass, getClassCodeByRoleID } from '../database/class-db';
+import { addChannelStringOption, addClassCodeStringOption } from '../common/commandHelper';
+import { addHomeworkChannel } from '../common/discordutil';
+import { ICommand } from '../types/command';
+
+const slashCommandBuilder = new SlashCommandBuilder();
+slashCommandBuilder.setName('addcc').setDescription('Add a class to the db.').setDefaultPermission(false);
+addClassCodeStringOption(slashCommandBuilder, true);
+addChannelStringOption(slashCommandBuilder, true)
+  .addRoleOption((option) => option.setName('role').setDescription('The role for the class').setRequired(true))
```

Comment:
we use `addRoleOption` frequently, this could be a helper too where we just pass in the description

(I'm loving the helpers btw)

Reply:
that is why i didn't go ahead and add it _yet_ 😆